src/source/Utils/ExponentialBackoffUtils.h

```diff
@@ -0,0 +1,139 @@
+/******************************************************
```

Comment:
In our codebase, all APIs with `PUBLIC_API ` are defined in `Include.h`. If all the functions/structure/#defines in this file are going to be accessible for customers to use, let us move those to `Include.h`. Only structures/functions that are used internally (not exposed to customer) are supposed to be kept in this file.

Reply:
+1

Reply:
I think this doesn't hold true now given that we are moving this piece to PIC. I'll remove  `PUBLIC_API` from the declaration.

Reply:
I think your functions need to have the` PUBLIC_API` tag even after moving to PIC, because atleast some of these functions are being used in the samples.